frontend/src/workStation/CodeGroup.jsx

```diff
@@ -0,0 +1,352 @@
+        options
+      );
+      setGroupdata(response.data);
+      setAllFiles(response.data.files);
+      setAllMembers(response.data.members);
+    } catch (error) {
+      setError("Error fetching group details");
+    }
+  };
+
+  useEffect(() => {
+    fetchGroupDetails();
+  }, []);
+
+  const handleUpdateGroupDetails = async () => {
+    try {
+      const response = await axios.put(
+        `${BACKEND_URL}/api/group/${groupId}/update-code-group`,
+        { newGroupName, newGroupImageUrl, newMembers },
+        options
+      );
+    } catch (error) {
+      setError("Error updating group details");
+    } finally {
+      setNewGroupName("");
+      setNewGroupImageUrl("");
+      setNewMembers([]);
+      fetchGroupDetails();
+      onUpdateGroupModalClose();
+      onAddMembersModalClose();
+    }
+  };
+
+  function handleAddNewMemberToList() {
+    if (newMemberInput) {
+      setNewMembers([...newMembers, newMemberInput]);
+      setNewMemberInput("");
+    }
+  }
+
+  function handleRemoveNewMemberToList(memberUsername) {
+    setNewMembers(newMembers.filter((member) => member !== memberUsername));
+  }
+
+  const handleCreateNewFile = async () => {
+    try {
+      const response = await axios.post(
+        `${BACKEND_URL}/group/${groupId}/create-new-file`,
+        { newFileName },
+        options
+      );
+    } catch (error) {
+      setError("Error Creating Group. Try again");
+    } finally {
+      setNewFileName("");
+      fetchGroupDetails();
+      onCreateNewFileModalClose();
+    }
+  };
+
+  function handleOpenFile(fileId) {
+    navigate(`/group/${groupId}/files/${fileId}/workstation`);
+  }
+
+  const handleDeleteFile = async (fileId) => {
+    try {
+      await axios.delete(
+        `${BACKEND_URL}/group/${groupId}/files/${fileId}/delete-file`,
+        options
+      );
+      fetchGroupDetails();
+      alert(`File deleted`);
+    } catch (error) {
+      setError("Error deleting file");
+    }
+  };
+
+  return (
+    <Box>
+      <Nav />
+      <Box>{groupData.groupName}</Box>
+      <img
+        src={groupData.imgUrl}
+        alt={`${(groupData.groupName, "'s image")}`}
+      />
+      <Button mt={4} className="btn" onClick={onUpdateGroupModalOpen}>
+        Update Group
+      </Button>
+      <HStack mt={4}>
+        <Box
+          w="50%"
+          id="code-group-files"
+          height="75vh"
+          p={2}
+          border="1px solid"
+          borderRadius={4}
+        >
+          <Box m={4}>
+            <ul>
+              {allFiles?.length > 0
+                ? allFiles.map((file, index) => {
+                    return (
+                      <li key={index}>
+                        <span onClick={() => handleOpenFile(file.id)}>
+                          {file.fileName} &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                        </span>
+                        <span onClick={() => handleDeleteFile(file.id)}>
+                          delete file
```

Comment:
nit: You should have consistent capitalization. In some places you use sentence case and in others you use no capitalization. I would recommend sticking with sentence case.

Reply:
Thanks. Fixed here: https://github.com/izuchukwumba/peered/commit/a960d993a9510bf0b293543fdb359403c740e67f